### PR TITLE
Add documentation for gluster-ansible-cluster role

### PR DESCRIPTION
**Pull request by devyanikota**
Signed-off-by: Devyani Kota <devyanikota@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
-Ansible role to create and deploy a Gluster cluster.
+gluster.cluster
+===============
+
+This role helps the user to set up a GlusterFS cluster, manage gluster volumes and peer operations.
+The gluster.cluster role has multiple sub-roles which are invoked depending on the variables that are set.
+The sub-roles are:
+
+  1. gluster_volume - manage gluster volumes(create, delete, start, stop)
+  2. gluster_volume_set - set specific options for gluster volumes
```
Comment (sac) [inline README.md]: @devyanikota can't this be part of volume? That is if gluster_volume has state started/stopped and the options variables are set then the volume would set those options and leave the volume in the said state.

A separate gluster_volume_set  role would be extra work.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+  3. gluster_brick - perform add/remove/replace operations on existing bricks
```
Comment (sac) [inline README.md]: Same for this.

**Review by sac — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (devyanikota) [reply README.md]: sure, will rectify this.

**Review by sac — APPROVED** [review]
LGTM

**Pull request merged**